### PR TITLE
Update to borgmatic 1.7.8

### PR DESCRIPTION
**Pull request by grantbevis**
Ugrade borgmatic to 1.7.8 and stop compiling `supercronic`

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -14,28 +9,29 @@ VOLUME /root/.config/borg
 VOLUME /root/.ssh
 VOLUME /root/.cache/borg
 RUN apk add --update --no-cache \
-    tzdata \
-    sshfs \
-    openssl1.1-compat \
-    fuse \
+    bash \
```
Comment (grantbevis) [inline base/Dockerfile:12]: This is only because I sorted the packages alphabetically to make it easier to visually grep

File: base/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,6 @@
 # syntax = docker/dockerfile:latest
-FROM golang:1.20.0-alpine3.17 AS gobuilder
-ARG SUPERCRONIC_VERSION="v0.2.1"
-RUN apk add --no-cache git
-RUN go install github.com/aptible/supercronic@"${SUPERCRONIC_VERSION}"
-
-FROM python:3.11.1-alpine3.17
-LABEL maintainer='b3vis'
```
Comment (grantbevis) [inline base/Dockerfile:8]: Not required as now doing a `apk add ... supercronic` in the RUN below

**Comment by grantbevis** [comment]
This will also close #213

**Comment by modem7** [comment]
LGTM!

**Review by modem7 — APPROVED** [review]
LGTM

**Review by modem7 — APPROVED** [review]
LGTM

**Pull request merged**